build/Dotty/packageInfo.json

```diff
@@ -178,5 +178,11 @@
     },
     {
         "packageName": "system.data.odbc"
+    },
+    {
+        "packageName": "opentelemetry"
+    },
+    {
+        "packageName": "opentelemetry.exporter.opentelemetryprotocol"
```

Comment:
We can leave this in for now and see how well it works, but we may decide to not use this approach because we may not want to update to some of the preview releases.